ts-pkgs/peer-e2e/tests/e2e/scripts/anvil.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -meuo pipefail
+export DOCKER_BUILDKIT=1
+
+docker build -t anvil-with-verifier -f ./anvil/Dockerfile --progress=plain ./anvil
```

Comment:
Let's prefer long options for docker.
```suggestion
docker build --tag anvil-with-verifier --file ./anvil/Dockerfile --progress=plain ./anvil
```